app/src/pages/Instructions/Instructions.tsx

```diff
@@ -55,138 +67,76 @@ const Instructions = () => {
         <HvCard statusColor="positive" bgcolor="atmo1">
           <HvCardHeader
             title={
-              <HvBox className={classes.title}>
+              <div className="flex flex-row items-center">
                 <Tool iconSize="M" />
                 <HvTypography variant="title1">Using the Editor</HvTypography>
-              </HvBox>
+              </div>
             }
           />
           <HvCardContent>
-            <HvTypography variant="title2" className={classes.subSection}>
+            <HvTypography variant="title2" className="mb-sm">
               Tools
             </HvTypography>
-            <HvBox className={classes.tool}>
-              <HvBox className={classes.toolTitle}>
```

Comment:
The refactor of these `classes.tool` and `classes.toolTitle` leads to multiple repeated classes, which means that a [refactor is in order](https://tailwindcss.com/docs/reusing-styles#extracting-components-and-partials), hence the `Entry` component, reducing duplication